docs/Initial-OnPrem-Setup/Setting-up-your-On-Premise-Machine.rst

```diff
@@ -0,0 +1,196 @@
+Next, install the U250 FPGA as indicated: https://docs.xilinx.com/r/en-US/ug1301-getting-started-guide-alveo-accelerator-cards/Card-Installation-Procedures
+
+We require the following programs/packages installed from the Xilinx website in addition to a physical U250 installation:
+
+* Xilinx Vitis 2022.1
+  * Installation link: https://www.xilinx.com/products/design-tools/vitis/vitis-whats-new.html#20221
+* Xilinx XRT and U250 board package (corresponding with Vitis 2022.1)
+  * Ensure you complete the "Installing the Deployment Software" and "Card Bring-Up and Validation" sections in the following link: https://docs.xilinx.com/r/en-US/ug1301-getting-started-guide-alveo-accelerator-cards/Installing-the-Deployment-Software
+
+Setup Validation
+~~~~~~~~~~~~~~~~
+
+After installing the U250 FPGA using the Xilinx instructions and installing the specific versions of Vitis/XRT, let's verify that the U250 FPGA can be used for emulations.
+Ensure that you can run the following XRT commands without errors:
+
+.. parsed-literal::
+
+   xbutil examine # obtain the BDF associated with your installed U250 FPGA
+   xbutil validate --device <CARD_BDF_INSTALLED> --verbose
+
+The ``xbutil validate`` command runs simple tests to ensure that the FPGA can be properly flashed with a bitstream by using XRT.
+
+.. Warning:: Anytime the host computer is rebooted you may need to re-run parts of the setup process (i.e. re-flash the shell).
+     Before continuing to FireSim simulations after a host computer reboot, ensure that the previously mentioned ``xbutil`` command is successful.
+
+Finally, ensure that the XRT/Vitis tools are sourced in your shell setup (i.e. ``.bashrc`` and or ``.bash_profile``) so that any shell can use the corresponding programs.
+The environment variables should be visible to any non-interactive shells that are spawned.
+You can check this by ensuring that the output of the following command shows that the XRT/Vitis tools are present in the environment variables (i.e. ``XILINX_XRT``):
+
+.. parsed-literal::
+
+    ssh localhost printenv
+
+Other Miscellaneous Setup
+-------------------------
+
+Additionally, you should be able to run ``ssh localhost`` without needing a password.
+The FireSim manager program runs all commands by ``ssh``-ing into a BuildFarm/RunFarm machine given an IP address then running the command.
+To do so non-interactively it needs passwordless access to the machines to build/run on.
+In the case of this tutorial, our local machine or ``localhost``.
+
+Finally, if you are running this tutorial without ``sudo`` access you should also install the ``guestmount`` program and ensure it runs properly.
+This is needed by a variety of FireSim programs that mount disk images to copy in/out results of simulations.
+Most likely you will need to follow the instructions here to ensure ``guestmount`` doesn't error: https://askubuntu.com/questions/1046828/how-to-run-libguestfs-tools-tools-such-as-virt-make-fs-without-sudo.
```

Comment:
can you make "here" a link. Ending the URL with "." also seems like a potential problem.